### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by elgatovital**
Changes proposed in this pull request:
- Bump several libs to newest version

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -31,12 +31,11 @@ subprojects {
...
             android {
-                compileSdkVersion 30
-                buildToolsVersion "28.0.3"
+                compileSdkVersion 33
 
                 defaultConfig {
-                    minSdkVersion 19
-                    targetSdkVersion 30
+                    minSdkVersion 21
+                    targetSdkVersion 33
```
Comment (biafra23) [inline build.gradle:38]: Do we need to go to 33 here? It means we have tested the code with a 33 device. And that would be Android 13. Does @liliya-soroka  even have a device with Android 13?

I would prefer 30 here (Android 11)

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
```
Comment (biafra23) [inline gradle/wrapper/gradle-wrapper.properties:5]: LOL

**Review by biafra23 — APPROVED** [review]
Looks like the version bumps are the same as in the app, right?

File: build.gradle (hunk 1 above)
Comment (elgatovital) [reply build.gradle:38]: android app targets 33, also svalinn use is limited to ens resolution currently

**Review by biafra23 — APPROVED** [review]

**Pull request merged**